Gemfile

```diff
@@ -25,91 +25,93 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
+# Gems needed by other groups
+gem 'rake', :require => false
+
 group :development do
-  gem 'rake',                         :require => false
```

Comment:
Technically this change is probably correct. However it seems the whole file was changed.